dcrtimed/dcrtimed.go

```diff
@@ -1133,17 +1163,19 @@ func (d *DcrtimeStore) verifyV2(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Translate digest results.
-	var dReply v2.VerifyDigest
+	var dReply *v2.VerifyDigest = nil
```

Comment:
`= nil` is redundant.  The zero value of a pointer is already nil.  Also, what's the purpose of changing these to pointers?